### ISSUE TITLE
Feature Request: Display Filter for Notifications

### ISSUE THREAD

**Issue by gitmopp** [sni/Thruk#688]
If you open /thruk/cgi-bin/notifications.cgi?contact=all it will show all notifications in the defined timeperoid. If you change notifications.cgi?contact=MyUser it will show only notifications for the specified contact. It would be nice to have a web text field to change the name.
Or even better the same "Display Filter" we see in many other views.
Livestatus understands the Filter: service_description.
I tested the following query with an additional filter.
```
GET log
Columns: class time type state host_name service_description plugin_output message options state_type contact_name
Filter: service_description = ShowMyService
Filter: time >= 1488322800
Filter: time <= 1489100400
Filter: class = 3
And: 3
OutputFormat: json
ResponseHeader: fixed16
```

**Comment by sni** [comment]
Thats not as easy as it sounds. The logs table has a service_description, sure, but most other columns are different, so we cannot just use the usual filters.

**Comment by sni** [comment]
fixed with #692
